flare_proxy.py

```
@@ -0,0 +1,30 @@
+# noinspection PyUnresolvedReferences
+import chromedriver_binary  # Adds chromedriver binary to path
+import requests
+
+
+class FlaredSession:
+    def __init__(self, session: requests.Session):
+        self.session = session
+
+    def get(self, url, **kwargs):
```

Ruff (0.14.13)

Unused method argument: `kwargs`

(ARG002)